doc/query.md

```diff
@@ -506,6 +667,9 @@ Here:
     <strong>{</strong> &quot;$exists&quot; <strong>:</strong> <em>boolean</em> <strong>}</strong>
     <strong>{</strong> &quot;$type&quot; <strong>:</strong> <em>type</em> <strong>}</strong>  // not implemented
 
+<em>text-search-expression</em> :=     
+    <strong>{</strong> &quot;$text&quot; <strong>:</strong><strong> {</strong> &quot;$search&quot; <strong>:</strong> <em>string-value</em> <strong>}</strong> <strong>}</strong>
+
```

Comment:
Looks like you forgot to add `text-search-expression` to the `expression` clause?